workitem/workitemtype_repository.go

```diff
@@ -111,8 +112,9 @@ func ClearGlobalWorkItemTypeCache() {
 
 // Create creates a new work item in the repository
 // returns BadParameterError, ConversionError or InternalError
-func (r *GormWorkItemTypeRepository) Create(ctx context.Context, spaceID uuid.UUID, id *uuid.UUID, extendedTypeID *uuid.UUID, name string, description *string, icon string, fields map[string]FieldDefinition) (*WorkItemType, error) {
+func (r *GormWorkItemTypeRepository) Create(ctx context.Context, spaceID uuid.UUID, id *uuid.UUID, extendedTypeID *uuid.UUID, name string, description *string, icon string, fields map[string]FieldDefinition, categories []*uuid.UUID) (*WorkItemType, error) {
```

Comment:
@DhritiShikhar 

> > Can't we just add it later?

> Okay, So basically there is no "Update workitemtype" action. Also, WIT are associated with category during space template creation.

1. Yes, the space template will do that
1. Technically you're not changing the WIT itself when you want to attach a category to it. So why do you need an Update function on the WIT repository to create the association between a WIT and a category?